internal/kafka/internal/migrations/20221213120000_add_default_value_for_cluster_type_column.go

```diff
@@ -0,0 +1,27 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func addDefaultValueForClusterTypeColumn() *gormigrate.Migration {
+
+	return &gormigrate.Migration{
+		ID: "20221213120000",
+		Migrate: func(tx *gorm.DB) error {
+			if err := tx.Exec("ALTER TABLE clusters ALTER cluster_type SET DEFAULT 'managed'").Error; err != nil {
+				return err
+			}
+
+			if err := tx.Unscoped().Exec("UPDATE clusters set cluster_type='managed'").Error; err != nil {
```

Comment:
We can safely run this against all clusters as we do not have clusters of different types for the moment